src/routes/(app)/events/create/+page.svelte

```diff
@@ -0,0 +1,114 @@
+<script>
+  import PictureInput from '$lib/components/forms/picture-input.svelte';
+  import LabelInput from '$lib/components/forms/label-input.svelte';
+  import Button from '$lib/components/buttons/button.svelte';
+  import FormsHeader from '$lib/components/forms-header.svelte';
+</script>
+
+<div class="flex h-full flex-col">
+  <FormsHeader label="Novo Evento" />
+
+  <div class="flex w-[100] flex-grow flex-col justify-around md:flex-row">
+    <form
+      method="POST"
+      enctype="multipart/form-data"
+      class="flex flex-col font-semibold md:flex-row md:gap-64"
+    >
+      <div class="order-2 ml-5 mt-5 flex flex-col content-start justify-center gap-10 md:order-1">
+        <LabelInput
+          label="Título"
+          name="Title"
+          type="text"
+          placeholder="Insira o texto"
+          horizontal
+          textGap="25"
+          required="{true};"
+          className="flex-col ml-5 mr-5 md:flex-row md:ml-0 md:mr-0"
+        />
+        <LabelInput
+          label="Slug"
+          name="Slug"
+          type="text"
+          placeholder="Insira o texto"
+          horizontal
+          textGap="25"
+          required="{true};"
+          className="flex-col ml-5 mr-5 md:flex-row md:ml-0 md:mr-0"
+        />
+        <LabelInput
+          label="Início"
+          name="DateStart"
+          type="datetime-local"
+          placeholder="Insira o texto"
+          horizontal
+          textGap="25"
+          required="{true};"
+          className="flex-col ml-5 mr-5 md:flex-row md:ml-0 md:mr-0"
+        />
+        <LabelInput
+          label="Fim"
+          name="DateEnd"
+          type="datetime-local"
+          placeholder="Insira o texto"
+          horizontal
+          textGap="25"
+          required="{true};"
+          className="flex-col ml-5 mr-5 md:flex-row md:ml-0 md:mr-0"
+        />
+        <LabelInput
+          label="Descrição"
+          name="Description"
+          type="text"
+          placeholder="Insira o texto"
+          horizontal
+          textGap="25"
+          required="{true};"
```

Comment:
```suggestion
          required={true}
```
And same for the rest of them